core/src/themes/ios/default.tokens.ts

```diff
@@ -71,4 +79,52 @@ export const defaultTheme: DefaultTheme = {
     xxxl: 'var(--ion-radii-500)',
     xxxxl: 'var(--ion-radii-full)',
   },
+  components: {
+    IonActionSheet: {
+      buttonColorDisabled: 'var(--ion-color-gray-850)',
+      buttonBgSelected: 'var(--ion-color-gray-150)',
+    },
+    IonBreadcrumb: {
+      color: 'var(--ion-color-text-850)',
+      bgFocused: 'var(--ion-color-gray-50)',
+      iconColor: 'var(--ion-color-text-400)',
+      iconColorActive: 'var(--ion-color-text-850)',
+      iconColorFocused: 'var(--ion-color-text-750)', // Available only in iOS
+      indicatorBg: 'var(--ion-color-gray-100)',
+      indicatorBgFocused: 'var(--ion-color-gray-150)',
+      separatorColor: 'var(--ion-color-text-550)',
+    },
+    IonDatetime: {
+      bg: 'var(--ion-color-gray-950)',
+      timeBodyBg: 'var(--ion-color-gray-300)',
+    },
+    IonItem: {
+      paragraphTextColor: 'var(--ion-color-text-450)',
+      borderColor:
+        'var(--ion-components-ion-item-border-color, var(--ion-item-border-color, var(--ion-border-color, var(--ion-color-gray-250))))',
+    },
+    IonModal: {
+      handleBg: 'var(--ion-color-gray-350)',
+    },
+    IonPicker: {
+      highlightBg: 'var(--ion-color-text-150)', // Available only in iOS
+    },
+    IonRange: {
+      bg: 'var(--ion-color-gray-900)', // Available only in iOS
+    },
+    IonRefresher: {
+      nativeSpinnerColor: 'var(--ion-color-gray-450)', // Available only in iOS
+    },
+    IonSegmentButton: {
+      checkedIndicatorBg: 'var(--ion-color-gray-350)', // Available only in iOS
+    },
+    IonTabbar: {
+      bg: 'var(--ion-tab-bar-background-color, var(--ion-color-gray-50))',
```

Comment:
There are some components like this one that already have existing variables that are similar. Maybe we should either:
1. Leave it as in and deprecate them in v9 and remove them in v10.
2. Remove them now
3. Generate these component variables to not be in the `--ion-components-ion-{COMPONENT_NAME}-*`. Instead, generate them as `--ion-{COMPONENT_NAME}-*` that way we can re-use some of the existing variables.